### DISCUSSION TITLE
When can we start testing 0.14 with our cameras?

### DISCUSSION THREAD

**Discussion by mabed-fr** [blakeblackshear/frigate#9205] [General]
Hello,

When can we start testing 0.14 with our cameras?

Regards

**Comment by NickM-27** [comment]
the webui is under rapid development, there are many bugs, missing features, and literally everything is subject to change as we move through different ideas and decide on a final design. 

Once 0.13 is fully released AND 0.14 is merged in to the dev branch then builds will be automatically created and usable

**Reply by mabed-fr** [reply]
Great
